src/gui/core/event/dispatcher.hpp

```diff
@@ -623,7 +623,8 @@ class dispatcher
 		} else if constexpr(cat == event_category::text_input) {
 			return signal_text_input_queue_;
 		} else {
-			static_assert(utils::dependent_false_v<cat>, "No matching signal queue for category");
+			// "No matching signal queue for category"
+			utils::static_assert_false();
```

Comment:
This is a negative change, as it loses the error message…

Reply:
It was either this or remove it entirely, since Clang was rejecting the version that allowed an error message.

Reply:
Sounds like a clang bug. You're supposed to be able to use a message.

Reply:
See https://github.com/wesnoth/wesnoth/runs/6146018736?check_suite_focus=true#step:3:215

Reply:
That looks like it could have been solved with `decltype(cat)`.